packages/http/src/validator/HttpValidator.ts

```diff
@@ -0,0 +1,103 @@
+import { IValidation, IValidator } from '@stoplight/prism-core/types';
+import { IHttpConfig, IHttpRequest, IHttpResponse } from '@stoplight/prism-http/types';
+import { IHttpOperation } from '@stoplight/types';
+import { findResponseSpec } from './helpers/findResponseSpec';
```

Comment:
Separate with a space please. It is just a convention at stoplight to put all npm package imports at the top of a file, followed by a space and all file imports. Both sorted alphabetically.